### DISCUSSION TITLE
MoBi (11.3) Interface Issue

### DISCUSSION THREAD

**Discussion by wanyuchu** [Open-Systems-Pharmacology/Forum#1881] [Questions & Problems]
Dear all,

This morning, when I opened MoBi (version 11.3), I noticed that no panels (e.g., Building Blocks, Simulations) are visible (please see the attached screenshot). Clicking any button in the "Views" menu does not resolve the issue.

I have already tried restarting my laptop and reinstalling the OSP Suite. Also tried to send a project from PK-Sim to Mobi or directly start a new project in MoBi. But the problem persists. 

MoBi was working perfectly fine last week, and the PK-Sim interface is functioning normally.

Could anyone kindly guide me on how to resolve this issue?

Thank you in advance for your help! 

Best. 

Wendy
<img width="953" alt="mobi" src="https://github.com/user-attachments/assets/f2c2ea83-b648-41c1-83f5-61f921d968ca" />

**Comment by PavelBal** [comment]
Try this https://github.com/Open-Systems-Pharmacology/Forum/discussions/1150
